sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -1373,6 +1373,65 @@ abstract class DDLSuite extends QueryTest with DDLSuiteBase {
     }
   }
 
+  test("Clustering columns should match when appending to existing data source tables") {
```

Comment:
shall we put it in `DataFrameReaderWriterSuite`?

Reply:
sure, moved there.